aws-iot-provisioningtemplate/src/main/java/com/amazonaws/iot/provisioningtemplate/CreateHandler.java

```diff
@@ -115,13 +119,17 @@ private static ProvisioningHook getPreProvisioningHook(final ResourceModel model
         } catch (final ResourceAlreadyExistsException e) {
             throw new CfnAlreadyExistsException(ResourceModel.TYPE_NAME, templateName);
         } catch (final InvalidRequestException e) {
-            throw new CfnInvalidRequestException(templateRequest.toString(), e);
+            throw new CfnInvalidRequestException(e.getMessage(), e);
         } catch (final LimitExceededException e) {
             throw new CfnServiceLimitExceededException(ResourceModel.TYPE_NAME, e.toString());
         } catch (final InternalException e) {
```

Comment:
do we have to fix this as well? please check all the places we using toString on exception which is incorrect. We better to switch to getMessage() if it requires a string otherwise pass the exception object along.

Reply:
`getMessage()` is the right approach. Checked other places as well and this is the only place where we use `toString()` instead of `getMessage()`. Updated it